doc/source/rllib/rllib-offline.rst

```diff
@@ -5,437 +5,2081 @@
+    for i, row in enumerate(ds.iter_rows(prefetch_batches=10)):
+        # If the episode isn't terminated nor truncated, buffer the data.
+        if not row["terminateds"] and not row["truncateds"]:
+            eps_obs.append(row["obs"])
+            eps_actions.append(row["actions"])
+            eps_rewards.append(row["rewards"])
+            eps_extra_model_outputs["action_dist_inputs"].append(row["action_dist_inputs"])
+            eps_extra_model_outputs["action_logp"].append(row["action_logp"])
+        # Otherwise, build the episode.
+        else:
+            eps_obs.append(row["new_obs"])
+            episode = SingleAgentEpisode(
+                id_=row["eps_id"],
+                agent_id=row["agent_id"],
+                module_id=row["module_id"],
+                observations=eps_obs,
+                # Use the spaces from the environment.
+                observation_space=env.observation_space,
+                action_space=env.action_space,
+                actions=eps_actions,
+                rewards=eps_rewards,
+                # Set the starting timestep to zero.
+                t_started=0,
+                # You don't want to have a lookback buffer.
+                len_lookback_buffer=0,
+                terminated=row["terminateds"],
+                truncated=row["truncateds"],
+                extra_model_outputs=eps_extra_model_outputs,
+            )
+            # Store the ready-to-write episode to the episode buffer.
+            episodes.append(msgpack.packb(episode.get_state(), default=mnp.encode))
+            # Clear all episode data buffers.
+            eps_obs.clear()
+            eps_actions.clear()
+            eps_rewards.clear()
+            eps_extra_model_outputs = defaultdict(list)
+
+        # Write episodes to disk when the episode buffer holds 50 episodes.
+        if len(episodes) > 49:
+            # Generate a Ray dataset from episodes.
+            episodes_ds = data.from_items(episodes)
+            # Write the Parquet data and compress it.
+            episodes_ds.write_parquet(
+                f"/tmp/test_converting/file-{i}".zfill(6),
+                compression="gzip",
+            )
+            # Delete the dataset in memory and clear the episode buffer.
+            del episodes_ds
+            episodes.clear()
+
+    # If we are finished and have unwritten episodes, write them now.
+    if len(episodes) > 0:
+        episodes_ds = data.from_items(episodes)
+        episodes_ds.write_parquet(
+            f"/tmp/test_converting/file-{i}".zfill(6),
+            compression="gzip",
+        )
+        del episodes_ds
+        episodes.clear()
+
+Using Old API Stack ``SampleBatch`` Recordings
+----------------------------------------------
+If you have expert data previously recorded using RLlib's older API stack, it can be seamlessly utilized in the new Offline RL API by setting ``input_read_sample_batches=True``. Alternatively, 
```

Comment:
```suggestion
If you have expert data previously recorded using RLlib's old API stack, it can be seamlessly utilized in the new stack's Offline RL API by setting ``input_read_sample_batches=True``. Alternatively, 
```